### PR TITLE
He2019momentum+elampietti

### PR DESCRIPTION
**Pull request by elampietti**

### CODE REVIEW

File: _summaries/he2019momentum_1.md [hunk 1]
```diff
@@ -1,19 +1,87 @@
...
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Traditionally, computer vision tasks are more effective with supervised pre-training while tasks in natural language processing (NLP) are successful with unsupervised representation learning.
+This is because NLP tasks build tokenized dictionaries in their discrete signal spaces that unsupervised learning can be applied to.
+However, computer vision has a continuous, high-dimensional space that makes it more challenging for dictionary building.
+This paper focuses on how to build dynamic dictionaries for unsupervised visual representation learning using their Momentum Constrast (MoCo) for training.
```
Comment (patelojas) [inline _summaries/he2019momentum_1.md:13]: I like the way you presented the problem. Made it very easy to understand.

File: _summaries/he2019momentum_1.md [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
+Contrastive learning is used to train the encoder for a dictionary look-up task.
+This is done by finding the key that has the smallest contrastive loss with the query, meaning it is the most similar.
+This similarity loss function is shown below.
+
+![contrastive_loss](https://user-images.githubusercontent.com/7085644/140273552-d1d51632-da7f-4f34-bdd8-19a91c9e7cc0.PNG)
+
+Momentum Contrast is employed to make sure that the dynamic dictionary being built is large and the encoder of the keys is consistent during its evolution.
+This is done by using a queue for the dictionary data samples so that the dictionary size can be larger than the mini-batch size and set as a hyper-parameter.
+A momentum update is then employed during back-propagation to solve the issue for updating keys when using a queue for samples.
+This means that the encoder will evolve slowly, therefore keeping it consistent.
```
Comment (patelojas) [inline _summaries/he2019momentum_1.md:34]: For this section, I would maybe expand each subsection (dictionary as  a queue, momentum update) a bit and give some more description on what is actually being done.

Comment (patelojas) [reply _summaries/he2019momentum_1.md:34]: Maybe also add some content from "Relations to previous mechanisms"

File: _summaries/he2019momentum_1.md [hunk 3]
```diff
@@ -1,19 +1,87 @@
...
+![VOC_results_2](https://user-images.githubusercontent.com/7085644/140277998-3b55cf93-a2f8-442e-96a3-18da9bf20fe0.PNG)
+
+Here are the results for the COCO dataset showing that MoCo is better than the supervised counterpart for both backbones.
+
+![COCO_results](https://user-images.githubusercontent.com/7085644/140278250-957a2f3a-652c-4551-8849-67da4a24acdd.PNG)
+
+Finally, downstream task evaluation reveals that MoCo is competitive with ImageNet supervised pre-training.
+
+![downstream_evaluation](https://user-images.githubusercontent.com/7085644/140278379-59d5c698-4c63-4d7b-84d9-c641b941d64c.PNG)
+
```
Comment (patelojas) [inline _summaries/he2019momentum_1.md:81]: Good summary of the results

File: _summaries/he2019momentum_1.md [hunk 4]
```diff
@@ -1,19 +1,87 @@
...
+The better results with MoCo pre-trained on the less curated dataset IG-1B compared to IN-1M shows that MoCo is good for real-world unsupervised learning.
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Unsupervised visual learning is challenging due to the continuous, high-dimensional space.
+* A dynamic dictionary that is large and consistent can be built using Momentum Contrast for unsupervised visual learning.
+* A queue of data samples for the dictionary allows for a large dictionary size while a momentum update allows for slow evolution and therefore consistency.
```
Comment (patelojas) [inline _summaries/he2019momentum_1.md:87]: Good summary. Went over the key points in the paper in an easy to understand format. I think if anything, it might be worth to add a few notes from the appendix, like A.8 and A.9 for talking about variations and ablations.

**Review by zhaoyue-zephyrus — APPROVED** [review]
